resources/views/search.blade.php

```diff
@@ -42,13 +37,13 @@
             <?php echo $wflow['description'] ?>
           </ul>
           <br />
-          Model Creator: <b>{{ $wflow['first_name'] }} {{ $wflow['last_name'] }}</b><br /> 4.5/5 Stars  |  <i><a href="/feedback">Provide Feedback</a></i>
+          Model Creator: <b><?php echo ($wflow['fname'] . ' '. $wflow['lname']) ?></b><br />
         </li>
       <?php endforeach;
     endif;
     if($numResult == 0): ?>
         <div class="alert alert-warning">No results found for : <?php echo $_GET['search']; ?> </div>
   <?php endif; ?>
 </div>
-
+<link href="{{ asset('css/app.css') }}" rel="stylesheet">
```

Comment:
CSS files should be put on top of the page (but scripts should be on the bottom)